### PR TITLE
Typos correction and addendum.

### PR DESCRIPTION
**Pull request by nbsn2**
Added Int128 Fp128 correction.

### CODE REVIEW

File: src/Llvm.NET/Native/Generated.cs [hunk 1]
```diff
@@ -866,13 +866,13 @@ internal static partial class NativeMethods
...
-        internal static extern LLVMTypeRef LLVMX8FP80Type( );
+        internal static extern LLVMTypeRef LLVMX86FP80Type( );
 
         [DllImport( LibraryPath, EntryPoint = "LLVMFP128Type", CallingConvention = CallingConvention.Cdecl )]
-        internal static extern LLVMTypeRef LLVMFP18Type( );
+        internal static extern LLVMTypeRef LLVMFP128Type( );
 
         [DllImport( LibraryPath, EntryPoint = "LLVMPPCFP128Type", CallingConvention = CallingConvention.Cdecl )]
-        internal static extern LLVMTypeRef LLVMPPCFP18Type( );
+        internal static extern LLVMTypeRef LLVMPPCFP128Type( );
```
Comment (smaillet) [inline src/Llvm.NET/Native/Generated.cs:876]: Let's just delete the variants that don't take a context, as they aren't/shouldn't be used by the LLVM.NET projections. The default context is dangerous to use or assume in multi-threaded programming and creating a new context is as simple as `new Context()`. The P/Invoke calls are being moved directly into the classes using them so we'll be dropping all the others here as well as we go forward on that.

**Comment by nbsn2** [comment]
Seems like kind of "trick of notes"? 
So just let it be and wait for further work.

**Comment by smaillet** [comment]
>Seems like kind of "trick of notes"?

Not sure I understand that question. Or why you chose to close this.

There are two variants of each of the basic type creation APIs. Basically along the lines of LLVMxyzType() and LLVMxyzTypeInContext(), My comment was about dropping the LLVMxyzType() forms. The InContext() variant should still remain (and be correctly spelled). 

A natural follow up to this change is to then update Llvm.NET.Context class to include accessor properties for these specialized types. That should be a relatively straight forward task using the existing primitive type implementations as a guide.

**Comment by smaillet** [comment]
These changes are now part of the llvm_60 branch

**Pull request closed without merging**